db/migrate/20190617191109_move_awx_credentials_to_authentications.rb

```diff
@@ -139,6 +139,9 @@ def self.decrypt(field_name, value, primary_key)
       return value unless value.include?("$encrypted$")
 
       new(secret_key, value, field_name, primary_key).decrypt
+    rescue Fernet256::InvalidToken
```

Comment:
Is this a very specific class exception?  Is there a higher level invalid token class we can use?

Reply:
@jrafanie look higher up in the file 😉 

https://github.com/ManageIQ/manageiq-schema/blob/a3a2dcfe7cd63e65b3b7ba70634247519183408e/db/migrate/20190617191109_move_awx_credentials_to_authentications.rb#L23

Reply:
Oh wow, I forgot we have that implemented here.  👍